src/main/java/org/jenkinsci/plugins/workflow/job/properties/PipelineTriggersJobProperty.java

```diff
@@ -50,16 +50,19 @@
 import java.util.List;
 import java.util.Map;
 
+@SuppressWarnings("rawtypes")
 public class PipelineTriggersJobProperty extends JobProperty<WorkflowJob> {
     private List<Trigger<?>> triggers = new ArrayList<>();
 
     @DataBoundConstructor
-    public PipelineTriggersJobProperty(List<Trigger<?>> triggers) {
+    public PipelineTriggersJobProperty(List<Trigger> triggers) {
         // Defensive handling of when we get called via {@code Descriptor.newInstance} with no form data.
         if (triggers == null) {
             this.triggers = new ArrayList<>();
         } else {
-            this.triggers = triggers;
+            for (Trigger t : triggers) {
+                this.triggers.add((Trigger<?>)t);
+            }
```

Comment:
Because `this.triggers = triggers;` doesn't work when `this.triggers` is `List<Trigger<?>>` and `triggers` is `List<Trigger>`. :hooray: